coderd/database/models.go

```diff
@@ -2093,6 +2093,8 @@ type NotificationMessage struct {
 	LeasedUntil            sql.NullTime              `db:"leased_until" json:"leased_until"`
 	NextRetryAfter         sql.NullTime              `db:"next_retry_after" json:"next_retry_after"`
 	QueuedSeconds          sql.NullFloat64           `db:"queued_seconds" json:"queued_seconds"`
+	// Auto-generated by insert/update trigger, used to prevent duplicate notifications from being enqueued on the same day
+	DedupeHash sql.NullString `db:"dedupe_hash" json:"dedupe_hash"`
```

Comment:
Will this actually ever be null? It always gets executed, right? In my experience getting the 'right' type here is just a case of nudging the query slightly (e.g. via explicit type cast).

Reply:
All entries in the `notification_messages` table up to this point will have a `NULL` value for this column once the migration occurs.

We could update these values in the migration, but I'm concerned we'll already have duplicates and this will fail the migration.

I think this is accurate right now, and we're not using this field directly in code anywhere so it doesn't matter much.